### PR TITLE
feat: upgraded library versions, better support for Solace queues

### LINKED ISSUE #95: Update Default Versions
Should update the default versions of generated apps: 
* Use latest Spring Boot 2.4.X release. Since Spring Boot has modified their releases, 2.5 (when released) may deprecate code from 2.4 but won't remove it. By moving to 2.4 this should future proof the generated apps for a bit. 
* While doing that might as well update to latest Spring Cloud + default binder versions

### PR DESCRIPTION
**Pull request by damaru-inc**
**Description**

Libraries were upgraded as follows:
Solace Spring Cloud: 2.1.0
Spring Boot: 2.4.7
Spring Cloud: 2020.0.3
Spring Cloud Stream: 3.1.3.RELEASE

As well, better support has been added for Solace queues. If a queue has been specified, it becomes the channel (destination) and the appropriate topic subscriptions shall be added to the queue.
- ...

**Related issue(s)**
Fixes #95

### CODE REVIEW

**Comment by derberg** [comment]
I think it should be a `feat` as merge should trigger minor release due to changes in parameters

**Comment by derberg** [comment]
Please have a look at sonarcloud report about code smells

**Comment by damaru-inc** [comment]
> Please have a look at sonarcloud report about code smells

Fixed them.

**Comment by damaru-inc** [comment]
> I think it should be a `feat` as merge should trigger minor release due to changes in parameters

I agree. Fixed.

**Review by derberg — DISMISSED** [review]

File: hooks/post-process.js [hunk 1]
```diff
@@ -99,16 +90,16 @@ module.exports = {
     // This renames schema objects according to the title field. By default we won't do this, we might add this as an option.
 
     //const schemas = asyncapi.components().schemas();
-    //console.log("schemas: " + JSON.stringify(schemas));
+    //console.log('schemas: ' + JSON.stringify(schemas));
```
Comment (fmvilas) [inline hooks/post-process.js:93]: May be a good time to remove this? :)

Comment (damaru-inc) [reply hooks/post-process.js]: Yes, I agree. I removed it.

**Review by fmvilas — APPROVED** [review]

**Pull request merged**